blob-clients-common/blob-jersey2-client-common/src/main/java/com/bazaarvoice/emodb/blob/jersey2/client/AbstractBlobStoreJersey2ClientFactory.java

```diff
@@ -35,7 +39,32 @@ public boolean isRetriableException(Exception e) {
                 e instanceof JsonStreamingEOFException;
     }
     public BlobStore usingCredentials(final String apiKey) {
-        AuthBlobStore authBlobStore = new BlobStoreJersey2Client(_endPoint, _client, _connectionManagementService);
+        if (_retryPolicy == null)
+            createdDefaultRetryPolicy();
+        AuthBlobStore authBlobStore = new BlobStoreJersey2Client(_endPoint, _client,
+                _connectionManagementService, _retryPolicy);
         return new BlobStoreJersey2AuthenticatorProxy(authBlobStore, apiKey);
     }
+
+    private void createdDefaultRetryPolicy() {
```

Comment:
default policy should be extracted to some protected static method and include code mentioned in https://github.com/bazaarvoice/emodb/pull/437/files#diff-ff4b5d285b67f3d64476b4bc3e518fe60aa41587706968ae0e2ad47cf69f6e26R122-R139.

all places where the retry policy is not specified  should use this method